site/using-weave/service-management.md

```diff
@@ -38,6 +39,17 @@ With the above in place, you are ready to connect to the 'nc' service from anywh
 
 Similar NAT rules to the above can used to expose services not just to the outside world but also other, internal, networks.
 
+###<a name="export-route"></a> Exporting Services to the wider network
+
+If you want hosts that are not running Weave Net to access services
+exposed as above, on each of those machines:
```

Comment:
@rade the previous section does make the entire weave network accessible, from the local host.  But I get your point: my addition is not about "services exposed as above".